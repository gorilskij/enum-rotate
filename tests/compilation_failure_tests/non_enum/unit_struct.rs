use derive_enum_rotate::EnumRotate;

#[derive(EnumRotate)]
struct Struct;
