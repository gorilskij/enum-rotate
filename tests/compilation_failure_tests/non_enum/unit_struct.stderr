error: Struct Struct is not an enum, EnumRotate can only be derived for enums
 --> tests/compilation_failure_tests/non_enum/unit_struct.rs:4:1
  |
4 | struct Struct;
  | ^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compilation_failure_tests/non_enum/unit_struct.rs:4:15
  |
4 | struct Struct;
  |               ^ consider adding a `main` function to `$DIR/tests/compilation_failure_tests/non_enum/unit_struct.rs`
